exporters/otlp/otlpgrpc/options.go

```diff
@@ -123,23 +139,46 @@ func WithHeaders(headers map[string]string) Option {
 // these credentials can be done in many ways e.g. plain file, in code tls.Config
 // or by certificate rotation, so it is up to the caller to decide what to use.
 func WithTLSCredentials(creds credentials.TransportCredentials) Option {
-	return func(cfg *config) {
-		cfg.clientCredentials = creds
-	}
+	return otlpconfig.NewGRPCOption(func(cfg *otlpconfig.Config) {
+		cfg.Traces.GrpcCredentials = creds
+		cfg.Metrics.GrpcCredentials = creds
+	})
+}
+
+// WithTracesTLSCredentials allows the connection to use TLS credentials
+// when talking to the traces server. It takes in grpc.TransportCredentials instead
+// of say a Certificate file or a tls.Certificate, because the retrieving
```

Comment:
Similar for `WithMetricsTLSCredentials`

```suggestion
// of say a Certificate file or a tls.Certificate, because the retrieving of
```